tasks/push-disk-images/README.md

```diff
@@ -10,6 +10,10 @@ The environment to use is pulled from the `cdn.env` key in the data file.
 | snapshotPath             | Path to the JSON file of the Snapshot spec in the data workspace                          | No       | -             |
 | dataPath                 | Path to data JSON in the data workspace                                                   | No       | -             |
 | pipelineRunUid           | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       | -             |
+| resultsDirPath           | Path to results directory in the data workspace                                           | No       | -             |
+
+## Changes in 0.3.0
+* The task now writes pushed image details to a results json file in the workspace
```

Comment:
This isn't accurate. The snapshot is not what we pushed

Reply:
@johnbieren  keep in mind that this is the mapped snapshot. So while most of it is input, there is the target repo there. I think it's better to save the resulting artifacts as they are pushed, but on the other hand as long as the pipeline succeeds, this should be it, right? (But again, we don't need all the other data.)

Reply:
There is a lot of information even if the mapped snapshot that is not a release pipeline artifact. I don't want to claim them as such

Reply:
updated in https://github.com/konflux-ci/release-service-catalog/pull/717/commits/0fe1260c88ca28ef3175221cb6d9712f407131d9